docs/workshop/onboarding.md

```diff
@@ -23,7 +25,7 @@ Select login with `operate-first`, and login into the cluster by using your Gith
 
 2. **Ceph S3 storage -**
 
-Ceph provides a unified storage service with an object, block, and file interface from a single cluster built from commodity hardware components. Operate First hosts a public ceph storage bucket that is accessible by anyone. You can find the credentials and other information for this bucket [here](https://etherpad.opendev.org/p/workshop_scale_2022).
+Ceph provides a unified storage service with an object, block, and file interface from a single cluster built from commodity hardware components. Operate First hosts a public ceph storage bucket that is accessible by anyone. You can find the credentials and other information for this bucket [here](https://vault.bitwarden.com/#/send/zTA4PuNJwEW6kq7ZAUnY8g/pf51QZhZcEQ4QCEN7Lbszw).
 
```

Comment:
For a password to this vault, please check with the workshop organizers during the workshop.

Reply:
okay, will add that here